src/OpenColorIO/transforms/builtins/Displays.cpp

```diff
@@ -285,6 +329,46 @@ void RegisterAll(BuiltinTransformRegistryImpl & registry) noexcept
                             CIE_XYZ_D65_to_ST2084_P3_D65_Functor);
     }
 
+    {
+        auto CIE_XYZ_D65_to_ST2084_DCDM_D65_Functor = [](OpRcPtrVec & ops)
+        {
+            // Scale 1.0 in the reference space to 0.48. The PQ function will interpret 
```

Comment:
IS the scaling related to ACES2 `linear_scale_factor`? Because it is already applied in the [ACES2 builtins](https://github.com/remia/OpenColorIO/blob/aces2-output-transforms/src/OpenColorIO/transforms/builtins/ACES.cpp#L554). Sorry if I'm getting confused here.

Reply:
You are totally right, the scaling actually makes no sense here for several reasons.  Removed!